### PR TITLE
Refactor my-news controller to use v3

### PR DESCRIPTION
**Pull request by phamann**
As it was the last controller within the myft-page app still relying on v1. 

- Point to new `personalised-feed-api/v2` endpoint
- Refactor feed view model
- Refactor topic cards model
- Refactor tests 

/cc @adgad & @matthew-andrews 

## TODO
- [ ] Remove old `model/articev1` libraries
- [ ] Refactor email-generator to use new endpoint

### CODE REVIEW

**Comment by phamann** [comment]
@adgad This is now ready for review and merge

**Comment by adgad** [comment]
LGTM

File: server/controllers/my-news.js [hunk 1]
```diff
@@ -101,12 +102,11 @@ module.exports = function(req, res, next) {
...
 				viewModel.rawContent = content;
-				viewModel.content = content.slice(0, 50).map(articleContainer => {
-					articleContainer.content.notificationStatus = notificationStatii[articleContainer.userCompletion] || 'new';
-					articleContainer.content.primaryTags = articleContainer.signals.followed.tags;
-					return articleModelV1(articleContainer.content);
-				});
-
+				viewModel.content = content.slice(0, 50)
+										.map(content => Object.assign(v3toCard(content.content), {
+											notificationStatus: notificationStatii[content.userCompletion] || 'new',
```
Comment (adgad) [inline server/controllers/my-news.js]: I'm glad the word statii has stuck around. Probably one of my finer contributions of 2015.

Comment (phamann) [inline server/controllers/my-news.js]: :sunglasses:

**Pull request merged**